scala/src/test/scala/org/apache/spark/deploy/csharp/CSharpRunnerSuite.scala

```diff
@@ -26,4 +26,20 @@ class CSharpRunnerSuite extends SparkCLRFunSuite {
     assert(CSharpRunner.formatPath(path3).startsWith("."))
     assert(CSharpRunner.formatPath(path3).endsWith(path3))
   }
+
+  test("debug mode test without port number") {
+    val args = Array("debug")
+    val settings = CSharpRunner.initializeCSharpRunnerSettings(args)
+
+    assert(settings._1)
+    assert(CSharpRunner.MOBIUS_DEBUG_PORT == settings._2)
+  }
+
+  test("debug mode test with port number") {
+    val args = Array("debug", "1234")
+    val settings = CSharpRunner.initializeCSharpRunnerSettings(args)
+
+    assert(settings._1)
+    assert(1234 == settings._2)
+  }
```

Comment:
I was planning to add that but dropped the idea because that would exercise the same code path as "debug mode test with port number" and validation would be the same too